vehicle/raw_vehicle_cmd_converter/include/raw_vehicle_cmd_converter/accel_map.h

```diff
@@ -22,22 +22,24 @@
 #include <string>
 #include <vector>
 
-#include <ros/ros.h>
+#include <rclcpp/rclcpp.hpp>
 
 #include "raw_vehicle_cmd_converter/csv_loader.h"
 #include "raw_vehicle_cmd_converter/interpolate.h"
 
 class AccelMap
 {
 public:
-  AccelMap();
+  AccelMap(const rclcpp::Logger & logger);
   ~AccelMap();
 
   bool readAccelMapFromCSV(std::string csv_path);
   bool getThrottle(double acc, double vel, double & throttle);
   bool getAcceleration(double throttle, double vel, double & acc);
 
 private:
+  rclcpp::Logger logger_;
```

Comment:
Looks like you are copying a logger. Is that OK in ROS2; i.e. do you still log to the same sink when you use the copy?

Reply:
Good point.
After you pointing that out, I searched a little bit, and it looks like the logger_usage demo provided by ROS2 also copies a logger so I think it's alright.
I've also checked by running the node without giving valid path for accel/brake map, and I was able to see that error message coming out on terminal.

Reply:
ok resolved